x/programs/rust/examples/counter-external/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+use wasmlanche_sdk::{public, types::Address, Context, ExternalCallContext, Program};
+
+#[public]
+pub fn inc(_: Context, external: Program, address: Address) {
+    let ctx = ExternalCallContext::new(external, 1_000_000);
+    // TODO: currently need to specify the return type (not good)
+    counter::inc(ctx, address, 1);
+}
+
+#[public]
+pub fn get_value(_: Context, external: Program, address: Address) -> u64 {
+    let ctx = ExternalCallContext::new(external, 1_000_000);
+    counter::get_value(ctx, address)
+}
+
+#[cfg(test)]
+mod tests {
+    use simulator::{ClientBuilder, Endpoint, Key, Param, Step};
+    use std::error::Error;
+
+    const PROGRAM_PATH: &str = env!("PROGRAM_PATH");
+
+    #[test]
+    fn inc_and_get_value() -> Result<(), Box<dyn Error>> {
+        let mut simulator = ClientBuilder::new().try_build().unwrap();
+
+        let counter_path = PROGRAM_PATH
+            .replace("counter-external", "counter")
+            .replace("counter_external", "counter");
```

Comment:
This isn't a great solution, but neither is hardcoding the wasm as a fixture. I think this actually needs to be a part of the `wasmlanche-sdk` integration tests. 

I think this is fine as a stop-gap for now, though.